src/styles/blocks/card.scss

```diff
@@ -0,0 +1,75 @@
+.card {
+  max-width: 199px;
```

Comment:
The max-width of the card should be consistent with the design. If the design specifies a different width, it should be updated accordingly.